doc/mgmt/SONiC_Design_Doc_Unified_FRR_Mgmt_Interface.md

```diff
@@ -1066,4 +1064,43 @@ This enhancement to FRR-BGP Unified management framework does not disrupt data p
 
 # 9 Unit Test
 Testing of the configuration changes is manual and BGP configurations present in CONFIG_DB are converted to FRR configurations using config gen utility with help of Jinja template.
+
+## 9.1 Test case for v6only:
+When v6 is enabled on an BGP neighbor (interface), BGP session is establised on link local address only and not the IP address specified for the interface.
+
+Test CASE 1 : Configure BGP neighbor.
+
+|Command Executed | Result /Notes     |
+|:------------------|:-----------------|
+|sonic(config-router-bgp)# exit| |
+|sonic(config)# interface Ethernet 0||
+|sonic(config)# ipv6 enable||
+|sonic(conf-if-Ethernet0)# no shutdown| |
+|sonic(conf-if-Ethernet0)# ip address 1.1.1.2/30|The netmask is 30 or 31 |
+|sonic(conf-if-Ethernet0)# exit| |
+|sonic(config)# router bgp 101| |
+|sonic(config-router-bgp)# neighbor interface Ethernet 0| |
+|sonic(config-router-bgp-neighbor)# address-family ipv4 unicast| |
+|sonic(config-router-bgp-neighbor-af)# activate| |
+|sonic(config-router-bgp-neighbor-af)# exit| |
+|sonic(config-router-bgp-neighbor)# remote-as external||
+|sonic(config-router-bgp-neighbor)# do show bgp ipv4 unicast neighbors|BGP session is establised on IPV4|
+
+TEST CASE 2
+
+|Command Executed | Result      |
+|:------------------|:-----------------|
+|sonic(config-router-bgp-neighbor)# v6only| BGP session is re-established on Link local|
+
+
+TEST CASE 3
```

Comment:
Mention that v6only is getting disabled.

Reply:
@prvattem Sucheta is bringing the latest HLD from the community as part of this PR, she'll create a new PR for v6only feature.

Reply:
Can you send me the PR for v6only feature ?